src/utils/preprocessing.py

```diff
@@ -185,6 +185,10 @@ def standing_laying_data_preprocessing(source_path):
 
 
 def sample_systematic_from_artifacts(artifacts: list, n_artifacts: int) -> list:
+    '''
+    Code reference from cgm-ml
+    https://github.com/Welthungerhilfe/cgm-rg/blob/458bde6710758fc7c16331ad41e043b060b53c49/src/utils/preprocessing.py#L187
```

Comment:
Can you find a URL which doesn't contain `458bde6710758fc7c16331ad41e043b060b53c49`. Also I don't see this `preprocessing.py` in this location anymore.